upi/metal/terraform.tfvars.example

```diff
@@ -42,6 +42,15 @@ matchbox_trusted_ca_cert = "matchbox/tls/ca.crt"
 // The Project ID for Packet.net where servers will be deployed.
 packet_project_id = ""
 
+// The Packet.Net device plan slug.
+packet_plan = ""
+
+// The Packet.Net facilities code(s) to be used.
+packet_facility = ""
+
+// The UUID of the hardware reservation where you want this device deployed on Packet.net.
```

Comment:
It may be worth calling out here and in config.tf that the value "next_available" can be used instead of specifying a specific UUID.

Reply:
Fixed.